systemtest/src/test/java/io/strimzi/systemtest/matchers/IsLogHasUnexpectedErrors.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.matchers;
+
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+
+/**
+ * <p>A IsLogHasUnexpectedErrors is custom matcher to check log form kubernetes client
+ * doesn't have any unexpected errors. </p>
+ */
+public class IsLogHasUnexpectedErrors extends BaseMatcher<String> {
+
+    @Override
+    public boolean matches(Object actualValue) {
+        if (actualValue != null && actualValue instanceof String && !actualValue.equals("")) {
```

Comment:
1. `null` never passes an `instanceof` test, so you can always replace an expression like `foo != null && foo instanceof Bar` with plain `foo instanceofBar`.

2. But you can actually simplify this test further to just plain `if (!"".equals(actualValue)) {...` because `String.equals` will always return `false` for a `null` argument. (When comparing with a string literal always put the literal as the receiver, because a literal can never be `null`, so the expression can never NPE).

Reply:
I also thought about using `StringUtils` to have a code like `if (StringUtils.isBlank(actualValue)) {...}` but anyway current code should be improved